library/lib/core/application.cpp

```diff
@@ -221,6 +258,10 @@ void Application::quit()
 
 void Application::navigate(FocusDirection direction)
 {
+    // Dismiss navigation if input type was changed
```

Comment:
Is it actually like that in HOS?

Reply:
Yes, if view was selected by touch, first click on any navigation button will show the frame around that view without moving frame to next view.